osu.Game.Tests/Visual/Playlists/TestScenePlaylistsResultsScreen.cs

```diff
@@ -315,33 +291,23 @@ private IndexedMultiplayerScores createIndexResponse(IndexPlaylistScoresRequest
         {
             var result = new IndexedMultiplayerScores();
 
-            long startTotalScore = req.Cursor?.Properties["total_score"].ToObject<long>() ?? 1000000;
             string sort = req.IndexParams?.Properties["sort"].ToObject<string>() ?? "score_desc";
 
             for (int i = 1; i <= scores_per_result; i++)
             {
                 result.Scores.Add(new MultiplayerScore
                 {
-                    ID = currentScoreId++,
+                    ID = sort == "score_asc" ? --highestScoreId : ++lowestScoreId,
```

Comment:
All scores are forced to have 0 total score, because it's easier to handle this way. So we're relying on online ID to put scores in the correct order, and that's why a smaller score ID puts the score to the front of the list (it is assumed to have occurred earlier) and a larger score ID puts the score to the back of the list.